### ISSUE TITLE
Design Intent - Accessibility Feedback - IIR Team, Veteran Status Card, New Home & User Experience

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#94373]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- I think the overall goals here are great, and I'm excited to see some tandem web/app work happening!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Must:** Broadly speaking, there are two standards for how users may resize page content that we need to account for --- a user can zoom in the whole page (need to support up to 400% zoom, which is roughly the same as supporting a 320px-wide display), and a user can resize just the text of the page (need to support up to 200% increase, eg the largest font size available in Chrome under Settings - Appearance). Making the web and mobile app versions of the status card look like the printed card might box you in a bit when users start adjusting their settings. 
   
   I don't have a specific recommendation here, except that on a screen I'm not sure whether making it look like a card brings a ton of value on a screen, except as a preview for what it looks like when it's printed.
- [x] **Must:** Looking at the "View Proof of Veteran Status" element in the nametag ([example](https://www.figma.com/design/AobGKkBvIiDuYXR2QqkytC/IIR-Veteran-Status?node-id=990-55100&t=R8DPXwR52E1XKGqx-4)), what you have now is kind of button-y and kind of link-y. Since it navigates to a new page, it should definitely be a link. Some button and link interactions behave differently for assistive technology users, so we want to be careful to be visually clear and consistent around what an element is.
- [ ] **Should:** Thinking about Apple Wallet and Google Wallet, note that those apps may be available to users of more than just smartphones, so you'll want to account for that in your widescreen "desktop" designs. From a little searching, it looks like it's possible to open your Apple Wallet on MacOS and Google Wallet on a Chromebook (which then can sync with your mobile device). Just a reminder to be careful about any assumptions you make about users' devices.
- [ ] **Should:** The gradient background for the card meets WCAG AA color contrast, but it looks like left-most lighter blue color against white text is just shy of the 7:1 contrast ratio that would meet WCAG AAA. This is definitely not a must, but if it's possible to nudge that blue to a slightly darker shade that would give us a very satisfying AAA contrast across the whole thing.

### Mobile app notes
- [x] **Must:** Similar to the web side with the ability to increase font sizes and zoom, devices have the option for increased font sizes and the ability to zoom in on the devices. If display zoom is enabled, it increases the overall size of everything on the device (including font sizes). We’ve had issues with trying to do more complex layouts in React Native because of increased font sizes and display zoom (that can result in things breaking) which is why we’ve often made the choice to stack things vertically. This means that you could easily lose the card appearance and is why we opted to use the large panel for Veteran Status in the mobile app.
    - [ ] **Should:** We have different screen types and interactions than web and we’d probably recommend a modal like the [large panel](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Flagship%20design%20library/Templates/ScreenTypes#large-panel) instead of the hierarchical interaction they had. [See doc site](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Flagship%20design%20library/Templates/ScreenTypes) for more details.
- [x] **Must:** React Native does not allow for gradient backgrounds in the app. We initially planned to use a gradient background in the panel for Veteran Status and did try to incorporate it in a few different ways during the MVP, but we were ultimately unsuccessful.
- [x] **Must:** Consider how the design will translate to dark mode.
- [x] **Must:** Printing from the app is tricky. More likely path for a user would be the iOS / Android share sheet (examples of the [Android share sheet](https://m3.material.io/components/bottom-sheets/overview) and the [iOS share sheet](https://developer.apple.com/design/human-interface-guidelines/activity-views)), which includes options to print. _Note: Both iOS and Android / Material Design have UI kits in Figma that include the share sheet that you can use in your designs and could likely be implemented on the web side, as well._
- [x] **Should:** Consider having the app recognize which device you’re on & display only that OS “add to wallet” button (instead of both). We would also recommend seeing if this is something that is possible to detect on the web side, as well.
- [x] **Should:** Consider some of the requests from Veterans that we have received since launch. Some of these requests are:
    - **From BVA (Blinded Veterans Association):** Adding a [high-level disability indicator](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/7427). They felt strongly that the best place for this was within Veteran Status so that the person who was viewing their information for a discount would know immediately if they were blind, deaf, etc.
    - **From BVA:** [Adding the VHIC card](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/7426) to the app. They felt like the Veteran Status panel would be the best place to access it, but I didn't specifically include that in the ticket because our team had previously discussed doing discovery work to see if this was possible to do in a post-MVP.
    - **From BVA:** The ability to share their proof of Veteran status. I did not create a ticket for this, since this was something that our team had already planned to incorporate post-MVP. To us, adding that option felt like the mobile app equivalent of being able to print the card. Right now, a Veteran could print multiple copies of the card from the web and give them out to whoever they'd like or take a screenshot in their mobile app. Their reasoning was so that they could share it with their caregiver, family member, and/or spouse. They also mentioned online ordering. Some online retailers will send you a discount code if you contact them and tell them that you are a Veteran. They said being able to share their proof of Veteran Status in an email would be ideal. Otherwise, they have to try to scan in their VHIC card, BVA ID card, or take a screenshot in their VA portal (and hope it is accurate and showing the right information if they do).
    - Also, during recent research sessions for home screen personalization, a Veteran brought up / requested the possibility of being able to toggle information in the panel on and off (such as disability rating, DOB, etc.) so that he was only displaying information that was needed for the discount he was trying to receive.
    - We have received additional requests in our app store reviews. You can view them using our [Feedback Hub](https://feedback-hub-e659c24714b9.herokuapp.com/) or on [Github](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/va-mobile-app/reporting/App-Stores-Reviews). Here is one request: _"Solid app to check on claims. One improvement would be adding the type of discharge to the proof of veteran status."_
- **Should:** When thinking about success measures we would recommend checking with our team as we put some work into that, but it was shelved for MVP.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by DavidPearlUX** [comment]
Hi @briandeconinck ,
I went through and updated based on your feedback. Here's what I did:

- This version now supports compatibility with up to 400% zoom (320px-wide display) and supports up to a 200% text size increase.
- Replaced button with link in nametage
- Adopted vertical stacking to prevent layout breakage and optimized designs for increased font sizes and display zoom, while maintaining 'card' appearance. 
- Gradient backgrounds were removed due to React Native limitations
- All mobile app designs now fully accommodate dark mode
- Printing is no longer supported in this app version, focusing instead on device-native options like "add to wallet" buttons
- Still exploring options to enhance relevance, including potential privacy features like removing DOB and disability rating from the displayed information.

Here is the relevant figma link: https://www.figma.com/design/AobGKkBvIiDuYXR2QqkytC/IIR-Veteran-Status?node-id=1763-117757&t=h2nEtm7AsIYY6Ldj-1
